generators/entity-server/scripts/mfe-files.js.ejs

```diff
@@ -0,0 +1,32 @@
+const files = {
+    path: 'ui/widgets',
+    templates: [
+        <%_ files.map(file => { _%>
+        <%_ if (file.options.skip) { _%>
+        /*
+        <%_ } _%>
```

Comment:
Do we need the commented out section or can we remove it?

Reply:
My approach to this is to have the "skipped" files still visible somewhere for us to know they exist. Currently, tests are skipped as I did not found a good way to write those inside the .ejs.
In the end, there should be no "skipped" files in the file structure and these would not be in the generated file